Unknown command '--embedded' given

When I do `npm i emscripten-sdk` it causes an error:
```
npm ERR! code 1
npm ERR! path /Users/Jesper/repositories/narrow/www/node_modules/emscripten-sdk
npm ERR! command failed
npm ERR! command sh -c node src/postinstall.js
npm ERR! Already up to date.
npm ERR! Cloning into '/Users/Jesper/repositories/narrow/www/node_modules/emscripten-sdk/emsdk'...
npm ERR! Unknown command '--embedded' given! Type 'emsdk help' to get a list of commands.
npm ERR! /Users/Jesper/repositories/narrow/www/node_modules/cross-spawn-promise/lib/index.js:20
npm ERR!     var _err = new Error(`Exited with status ${code}`);
npm ERR!                ^
npm ERR! 
npm ERR! Error: Exited with status 1
npm ERR!     at closeArgsToError (/Users/Jesper/repositories/narrow/www/node_modules/cross-spawn-promise/lib/index.js:20:16)
npm ERR!     at ChildProcess.<anonymous> (/Users/Jesper/repositories/narrow/www/node_modules/cross-spawn-promise/lib/index.js:76:19)
npm ERR!     at Object.onceWrapper (node:events:485:26)
npm ERR!     at ChildProcess.emit (node:events:378:20)
npm ERR!     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12) {
npm ERR!   exitStatus: 1,
npm ERR!   stdout: null,
npm ERR!   stderr: null
npm ERR! }

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/Jesper/.npm/_logs/2021-03-29T13_26_45_888Z-debug.log
```

Not really sure why this started happening suddenly. If I remember correctly this was working fine for emscripten-sdk@2.0.13-1 but now it seems broken for that version as well.

And when I do `npx emsdk-checkout` I get:
```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/emsdk-checkout - Not found
npm ERR! 404 
npm ERR! 404  'emsdk-checkout@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/Jesper/.npm/_logs/2021-03-29T13_36_14_035Z-debug.log
```

not sure if this is related

--- a/src/emsdk.js
+++ b/src/emsdk.js
@@ -25,11 +25,10 @@ const common = require('./common.js');
 const version = require('./version.js');
 
 function emsdk(args, opts = {}) {
-    const emsdkArgs = ['--embedded'].concat(args);
     const emsdkDir = common.emsdkBase();
     const suffix = (process.platform === 'win32') ? '.bat' : '';
     const emsdk = path.join(emsdkDir, 'emsdk' + suffix);
-    return common.run(emsdk, emsdkArgs, opts);
+    return common.run(emsdk, args, opts);
 }
 
 if (require.main === module) {

--- a/src/emsdk.js
+++ b/src/emsdk.js
@@ -25,11 +25,10 @@ const common = require('./common.js');
 const version = require('./version.js');
 
 function emsdk(args, opts = {}) {
-    const emsdkArgs = args.concat('--embedded');
     const emsdkDir = common.emsdkBase();
     const suffix = (process.platform === 'win32') ? '.bat' : '';
     const emsdk = path.join(emsdkDir, 'emsdk' + suffix);
-    return common.run(emsdk, emsdkArgs, opts);
+    return common.run(emsdk, args, opts);
 }
 
 if (require.main === module) {

--- a/src/emsdk.js
+++ b/src/emsdk.js
@@ -25,10 +25,11 @@ const common = require('./common.js');
 const version = require('./version.js');
 
 function emsdk(args, opts = {}) {
+    const emsdkArgs = args.concat('--embedded');
     const emsdkDir = common.emsdkBase();
     const suffix = (process.platform === 'win32') ? '.bat' : '';
     const emsdk = path.join(emsdkDir, 'emsdk' + suffix);
-    return common.run(emsdk, args, opts);
+    return common.run(emsdk, emsdkArgs, opts);
 }
 
 if (require.main === module) {

